pkg/daemon/rpm-ostree.go

```diff
@@ -340,32 +329,95 @@ func (r *RpmOstreeClient) RebaseLayered(imgURL string) (changed bool, err error)
 	} else {
 		glog.Info("Current origin is not custom")
 	}
+}
 
-	glog.Infof("Executing rebase to %s", imgURL)
-	args := []string{"rebase", "--experimental", "ostree-unverified-registry:" + imgURL}
-
-	if err = runRpmOstree(args...); err != nil {
+// RebaseLayered potentially rebases system if not already rebased.
+func (r *RpmOstreeClient) RebaseLayered(imgURL string, pullSecret []byte) (err error) {
```

Comment:
Does it make more sense to pass pull secret to this function and handle writing here, or assume it's been written as a precondition?

Reply:
I think we should do this on daemon startup, and have the daemon sync loop update it.

Reply:
So that we can always assume any rpm ostree command will be able to use it?

Reply:
Right